### ISSUE TITLE
Overflow Menu: kebab dropdown in multiple groups and persistent examples inaccessible by keyboard

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3767]
**Describe the issue. What is the expected and unexpected behavior?**
When navigating by keyboard, you are able to press enter on the kebab and open the dropdown menu, but once there, you are unable to navigate to or interact with the menu items.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

<img width="658" alt="Screen Shot 2020-02-18 at 10 29 55 AM" src="https://user-images.githubusercontent.com/28281340/74750582-ec579e00-5239-11ea-81bd-b45ef9f4c2ee.png">

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This is an accessibility bug.
